.github/CODEOWNERS

```diff
@@ -621,3 +621,6 @@
 /singlestoredb_cloud/*metadata.csv     @tdasarathan @DataDog/ecosystems-review
 /singlestoredb_cloud/manifest.json     @tdasarathan @DataDog/ecosystems-review
 /singlestoredb_cloud/README.md         @tdasarathan @DataDog/ecosystems-review
+/dylibso-webassembly/manifest.json		@nilslice @bhelx @DataDog/ecosystems-review
+/dylibso-webassembly/README.md			@nilslice @bhelx @DataDog/ecosystems-review
```

Comment:
```suggestion
/dylibso-webassembly/                    @nilslice @bhelx @DataDog/ecosystems-review
```

Reply:
Apologies on this - I am working with my team to better understand the error. I believe it was actually due to the additional tab / space

Reply:
No problem -- will keep an eye out for any further instruction :)